configuration/etl_ddl/hiv_care_follow_up/hiv_follow_up.sql

```diff
@@ -0,0 +1,75 @@
+DELIMITER //
+
+DROP PROCEDURE IF EXISTS ssemr_etl_tables;
+
+-- Create the procedure
+CREATE PROCEDURE ssemr_etl_tables()
+BEGIN
+    DECLARE script_id INT(11);
+
+    DROP TABLE IF EXISTS ssemr.etl_script_status;
+
+    CREATE TABLE ssemr.etl_script_status
+    (
+        id          INT(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        script_name VARCHAR(50)  DEFAULT null,
+        start_time  DATETIME     DEFAULT NULL,
+        stop_time   DATETIME     DEFAULT NULL,
+        error       VARCHAR(255) DEFAULT NULL
+    );
+ 
+    INSERT INTO ssemr.etl_script_status(script_name, start_time) VALUES ('initial_creation_of_tables', NOW());
+    SET script_id = LAST_INSERT_ID();
+
+
+-- Create table hiv_care_follow_up --
+
+CREATE TABLE ssemr.ssemr_flat_encounter_hiv_care_follow_up
```

Comment:
@Michaelndula  please retain the "create table" statement only and remove DB name reference

Reply:
@moshonk well noted

Reply:
@moshonk I have removed the db name reference